xdp/src/netlink.rs

```diff
@@ -88,8 +136,10 @@ impl NetlinkSocket {
         Ok(())
     }
 
-    fn recv(&self) -> Result<Vec<NetlinkMessage>, io::Error> {
-        let mut buf = [0u8; 4096];
+    pub(crate) fn recv(&self) -> Result<Vec<NetlinkMessage>, io::Error> {
+        // create large buffer for when we poll the socket
+        // this helps us avoid any truncated messages when a single datagram contains many messages
+        let mut buf = [0u8; 64 * 1024];
```

Comment:
Increasing is probably fine, but the comment seems inaccurate? 

The buffer size shouldn't have anything to do with polling or not? And I
wouldn't expect anything to be truncated, just a multipart message split across
multiple recv(), which the code seems to be able to handle already?